### ISSUE TITLE
Install fails on cordova-android 7.1.1

### ISSUE THREAD

**Issue by jaco-terbraak** [apache/cordova-plugin-splashscreen#166]
Probably because the location of AndroidManifest.xml changed.

```
Installing "cordova-plugin-splashscreen" for android
Failed to install 'cordova-plugin-splashscreen': Error: ENOENT: no such file or directory, open '[redacted]/platforms/android/AndroidManifest.xml'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at Object.parseElementtreeSync ([redacted]/platforms/android/cordova/node_modules/cordova-common/src/util/xml-helpers.js:180:27)
    at new AndroidManifest ([redacted]/platforms/android/cordova/lib/AndroidManifest.js:29:20)
    at AndroidProject.getPackageName ([redacted]/platforms/android/cordova/lib/AndroidProject.js:99:12)
    at Api.addPlugin ([redacted]/platforms/android/cordova/Api.js:223:57)
    at handleInstall (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/plugman/install.js:594:10)
    at /usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/plugman/install.js:357:28
    at _fulfilled (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:854:54)
    at self.promiseDispatch.done (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:883:30)
ENOENT: no such file or directory, open [redacted]/platforms/android/AndroidManifest.xml'
```

**Comment by jaco-terbraak** [comment]
```
$ cordova -v
8.1.2 (cordova-lib@8.1.1)
```

**Comment by pedoc** [comment]
+1

**Comment by nevernet** [comment]
i got same issue, is there any news?

**Comment by janpio** [comment]
What version of the plugin are you installing?
Can you reproduce the problem in a new project created with `cordova create`?

**Comment by jaco-terbraak** [comment]
On a fresh project, it works fine. Going to look into what's wrong with my existing project...

**Comment by jcesarmobile** [comment]
This should be fixed in next cordova-android 7.1.2 release
